source/delegate-factory/package.json

```diff
@@ -27,9 +27,8 @@
     "verify": "truffle run verify"
   },
   "devDependencies": {
-    "@airswap/order-utils": "0.3.13",
-    "@airswap/test-utils": "0.1.3",
-    "solidity-coverage": "^0.6.3"
+    "@airswap/order-utils": "0.3.14",
+    "@airswap/test-utils": "0.1.3"
```

Comment:
do these updates require an update of the package.json version?

Reply:
Only if you change a dependency. If you change a `devDependency` then you dont need to redeploy the module. This is why `order-utils` has been bumped, as I changed the `dependency` section. The others only change `devDependency`.